build/rakelib/cluster.rake

```diff
@@ -119,6 +119,42 @@ namespace :cluster do
     end
   end
 
+  desc "Deploy ebrelayer"
+  namespace :ebrelayer do
+    task :deploy, [:chainnet, :provider, :namespace, :image, :image_tag, :ethereum_websocket_address, :ethereum_contract_address, :ethereum_private_key] do |t, args|
```

Comment:
we should update: `ethereum_contract_address` to : `eth_bridge_registry_address` for better naming